### PR TITLE
Add style guide

### PR DESCRIPTION
**Pull request by boniattirodrigo**

### CODE REVIEW

File: style-guide.md [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+    propA="a"
+    propB="b" />
+```
+
+### Method sequence
+1. Static methods
+2. `constructor`
+3. `componentWillMount`
+4. `componentDidMount`,
+5. `componentWilReceiveProps`,
```
Comment (willamesoares) [inline style-guide.md]: `ComponentWillReceiveProps`

File: style-guide.md [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+
+  render() {
+    // ...
+  }
+}
+```
+
+### Quotes
+
+Always use a single code, only over JSX components use double quotes.
```
Comment (willamesoares) [inline style-guide.md]: What about rephrasing this to: `Except for the JSX markup, you should always use double quotes.`

File: style-guide.md [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+
+Positioning comes first because it can remove an element from the normal flow of the document and override box model related styles. The box model comes next as it dictates a component's dimensions and placement.
+
+Everything else takes place inside the component or without impacting the previous two sections, and thus they come last.
+
+
+```javascript
+const styles = StyleSheet.create({
+  /* Positioning */
+  position: 'absolute',
```
Comment (willamesoares) [inline style-guide.md]: What about following alphabetical order for the props inside each block?

Comment (boniattirodrigo) [reply style-guide.md]: I don't think so, I guess there are some important properties that need to be in the top of stylization. We can do that in another PR.

**Pull request merged**